modules/restore-replace-stopped-baremetal-etcd-member.adoc

```diff
@@ -0,0 +1,592 @@
+----
+$ oc get nodes
+
+NAME           STATUS    ROLES    AGE    VERSION
+
+openshift-master-0 Ready master 3h24m v1.24.0+9546431
+openshift-master-1 Ready master 3h24m v1.24.0+9546431
+openshift-worker-0 Ready worker 176m v1.24.0+9546431
+openshift-worker-1 Ready worker 176m v1.24.0+9546431
+----
+
+. Create the new `BareMetalHost` object and the secret to store the BMC credentials:
+
++
+[source,terminal]
+----
+$ cat <<EOF | oc apply -f -
+apiVersion: v1
+kind: Secret
+metadata:
+  name: openshift-master-2-bmc-secret
+  namespace: openshift-machine-api
+data:
+  password: <password>
+  username: <username>
+type: Opaque
+---
+apiVersion: metal3.io/v1alpha1
+kind: BareMetalHost
+metadata:
+  name: openshift-master-2
+  namespace: openshift-machine-api
+spec:
+  automatedCleaningMode: disabled
+  bmc:
+    address: redfish://10.46.61.18:443/redfish/v1/Systems/1
+    credentialsName: openshift-master-2-bmc-secret
+    disableCertificateVerification: true
+  bootMACAddress: 48:df:37:b0:8a:a0
+  bootMode: UEFI
+  externallyProvisioned: false
+  online: true
+  rootDeviceHints:
+    deviceName: /dev/sda
+  userData:
+    name: master-user-data-managed
+    namespace: openshift-machine-api
+EOF
+----
++
+Note: username and password can be found from the other bmh's secrets. The protocol to use in bmc:address: can be taken from other bmh objects.
++
+[IMPORTANT]
+====
+If you reuse the `BareMetalHost` object definition from an existing control plane host, do not leave the `externallyProvisioned` field set to `true`.
+
+Existing control plane `BareMetalHost` objects may have the `externallyProvisioned` flag set to `true` if they were provisioned by the {product-title} installation program.
+====
++
+After the inspection is complete, the `BareMetalHost` object is created and available to be provisioned.
+
+. Verify the creation process using available `BareMetalHost` objects:
++
+[source,terminal]
+
+----
+$ oc get bmh -n openshift-machine-api
+
+NAME STATE CONSUMER ONLINE ERROR AGE
+openshift-master-0 externally provisioned ocp-edge-4zg89-master-0 true 4h48m
+openshift-master-1 externally provisioned ocp-edge-4zg89-master-1 true 4h48m
+openshift-master-2 provisioned ocp-edge-4zg89-master-3 true 47m
+openshift-worker-0 provisioned ocp-edge-4zg89-worker-0-8v2jj true 4h48m
+openshift-worker-1 provisioned ocp-edge-4zg89-worker-0-jb5z9 true 4h48m
+
+$ oc get machine -n openshift-machine-api
+
+NAME PHASE TYPE REGION ZONE AGE
+ocp-edge-4zg89-master-0 Running 4h49m
+ocp-edge-4zg89-master-1 Running 4h48m
+ocp-edge-4zg89-master-3 Running 25m
+ocp-edge-4zg89-worker-0-8v2jj Running 4h23m
+ocp-edge-4zg89-worker-0-jb5z9 Running 4h23m
+
+$ oc get nodes
+
+NAME STATUS ROLES AGE VERSION
+openshift-master-0 Ready master 4h26m v1.24.0+9546431
+openshift-master-1 Ready master 4h26m v1.24.0+9546431
+openshift-master-2 Ready master 12m v1.24.0+9546431
+openshift-worker-0 Ready worker 3h58m v1.24.0+9546431
+openshift-worker-1 Ready worker 3h58m v1.24.0+9546431
+----
```

Comment:
Please, delete rows 458-476. On this stage no machine and no node are created yet, only bmh is added in the state available

Reply:
Done, thank you!